GoTwitterCloneBackend/GoTwitterCloneApp/models/player.go

```diff
@@ -39,3 +49,21 @@ type PlayerCreateResponseModel struct {
 	Bio       string `json:"bio"`
 	CreatedAt int64  `json:"createdAt"`
 }
+
+type PlayerGetMinimumProfileResponseModel struct {
```

Comment:
identical to `PlayerCreateResponseModel`. should we just use one? idk. maybe it's best to keep them separate, just in case either one needs more fields that the other don't.